weepickle-tests/src/test/scala/com/rallyhealth/weepickle/v1/DefaultsTests.scala

```diff
@@ -63,11 +63,9 @@ object StringTests extends TestSuite {
   implicit val pickler = WeePickle.macroFromTo[A]
 
   override val tests: Tests = Tests {
-    test("write default")(FromScala(A(null)).transform(ToJson.string) ==> """{"d":null}""")
```

Comment:
What happened to these test cases? Would it make sense to update them to document and enforce the new behavior?

Reply:
I deleted these tests because they were testing scenarios where you were passing in a `null` to a case class, which doesn't seem like a use case worth supporting.

Reply:
Got it. I agree that using `null` isn't idiomatic/recommended usage.

However, there's history here (https://github.com/rallyhealth/weePickle/pull/43/files#r475258892). I'd prefer to keep these tests around to detect future behavior changes in this situation, particularly as we look toward scala 3 support and possibly borrowing macro code from upickle, which has different behavior here.

Reply:
I brought the tests back. This particular test is still being removed because it is identical to the "write null" test below.